### ISSUE TITLE
Add a column with IATI Organisation Identifiers for all IATI publishers

### ISSUE THREAD

**Issue by PetyaKangalova** [ViderumGlobal/ckanext-iati#137]
Could I check whether it will be possible to add a new column to [summary Publisher page](https://www.iatiregistry.org/publisher) and an estimate of the time involved to implement the change?

We would like to be able to see the IATI Organisation Identifier in a column for each publisher **after the publisher name.** See screenshot below

![image](https://user-images.githubusercontent.com/24607542/30641831-65c2b374-9e00-11e7-88af-6f6cba62338d.png)

 This issue is related to #94  which was asking about adding the IATI organisation identifier to the admin interface. If it is possible to add the IATI Organsation identifiers to the public interface, as requested in this issue, then no further action will be required on issue #94  and issue #94 can be closed.

**Comment by newkdukem** [comment]
This feature is added on the staging portal and waiting for approval.

**Comment by dalepotter** [comment]
@PetyaKangalova or @amy-silcock could you do a manual test for this to confirm if the functionality appears as intended?

**Comment by PetyaKangalova** [comment]
@dalepotter @newkdukem  I checked the list on [staging](https://iati-staging.ckan.io/publisher) and can confirm that the functionality appears as we have requested. I will be happy for this to be deployed to production.

One very small edit is to change the name from 

> IATI Organisation Identifier**s** 

to 

> IATI Organisation Identifier 

(remove the 's' as it refers to each individual publisher's identifier).

![image](https://user-images.githubusercontent.com/24607542/31889206-e0a3fb1a-b7f6-11e7-8f80-7dd752738017.png)
